### DISCUSSION TITLE
How to use the new File Picker styling

### DISCUSSION THREAD

**Discussion by Hendo16** [kikipoulet/SukiUI#241] [Q&A] [answered]
Hi, i've been working on an app that's implemented SukiUI from last year and had to recently take a break for a few months due to university. I'm dusting the project off and starting back up again and noticed the wonderful updates/progress that's been made to the library since i've been gone. Looks great!

In particular, I love the in-theme file picker - perfect for what i'm trying to do. But I can't seem to replicate this on my own app, in fact the code to open the file/folder picker in the demo is near identical to the code I already had to do the same thing. I'm struggling to understand the flow and what i need to change to implement this look - if anyone has any insights that'd be awesome!

**Comment by Hendo16** [comment]
I should mention that I have updated SukiUI to beta7, my avalonia is on 11.1.0-beta2 (as I think there's currently issues with rc2)

**Comment by kikipoulet** [comment, marked as answer]
Hi ! 

This option is still really in progress but can be used indeed, you just have to put "UseManagedSystemDialogs()" in your Program.cs file. You can check the Program.cs file of the SukiDemo project.
